tests/baselines/reference/deeplyNestedAssignabilityIssue.errors.txt

```diff
@@ -0,0 +1,67 @@
+[96mtests/cases/compiler/deeplyNestedAssignabilityIssue.ts[0m:[93m22[0m:[93m17[0m - [91merror[0m[90m TS2322: [0mType '{}' is not assignable to type 'A'.
+  Property 'a' is missing in type '{}'.
+
+[30;47m22[0m                 thing: {}
+[30;47m  [0m [91m                ~~~~~[0m
+
+  [96mtests/cases/compiler/deeplyNestedAssignabilityIssue.ts[0m:[93m8[0m:[93m19[0m
+    [30;47m 8[0m             more: {
+    [30;47m  [0m [96m                  ~[0m
+    [30;47m 9[0m                 thing: A;
+    [30;47m  [0m [96m~~~~~~~~~~~~~~~~~~~~~~~~~[0m
+    [30;47m10[0m             }
+    [30;47m  [0m [96m~~~~~~~~~~~~~[0m
+    Which is from property '"thing"' of type '{ thing: A; }', declared here.
+[96mtests/cases/compiler/deeplyNestedAssignabilityIssue.ts[0m:[93m25[0m:[93m17[0m - [91merror[0m[90m TS2322: [0mType '{}' is not assignable to type 'A'.
+  Property 'a' is missing in type '{}'.
+
+[30;47m25[0m                 another: {}
+[30;47m  [0m [91m                ~~~~~~~[0m
+
+  [96mtests/cases/compiler/deeplyNestedAssignabilityIssue.ts[0m:[93m11[0m:[93m23[0m
+    [30;47m11[0m             yetstill: {
+    [30;47m  [0m [96m                      ~[0m
+    [30;47m12[0m                 another: A;
+    [30;47m  [0m [96m~~~~~~~~~~~~~~~~~~~~~~~~~~~[0m
+    [30;47m13[0m             }
+    [30;47m  [0m [96m~~~~~~~~~~~~~[0m
+    Which is from property '"another"' of type '{ another: A; }', declared here.

```

Comment:
Because i'm stringifying the literal type associated with the name. It might equally be `'0'` or `[typeof foo]`. I can try to strip `"` from the type's string (if it's a valid identifier post-strip) if you think it'll improve the message.